Sources/TSCTestSupport/FileSystemExtensions.swift

```diff
@@ -72,3 +72,19 @@ extension FileSystem {
         }
     }
 }
+
+public extension AbsolutePath {
+    init(path: StaticString) {
```

Comment:
Without an argument label, the `String` version of the initializer seemed to always get chosen, so I ended up just adding an argument label. These initializers are a convenience for tests which often deal with static paths.